### DISCUSSION TITLE
Does passport work with serverless framework (aws lambda)?

### DISCUSSION THREAD

**Discussion by FerroO2000** [jaredhanson/passport#826] [Help]
Hi,

I really like the semplicity and the integrations with other services of passport. I am currently working on a website and I want to make it serverless. The questions are: has someone already used passport with serverless (aws lambda)? Is it possible?

Thanks

**Comment by philwilks** [comment]
@askflow1111 did you have any luck with this? I'm getting a timeout when the OAuth callback URL is called.
